### ISSUE TITLE
Issue related to laravel grid while mapping two tables

### ISSUE THREAD

**Issue by Manupreeta94** [leantony/laravel-grid#107]
Hello,
We are trying to join 2 tables from sql. We have some views and in one view we have list, which is to be displayed and which will have 2 tables joining. So when we create one model how can we call other table columns in list as grid command line creates list of columns from one model.

**Comment by basicAngular** [comment]
Same problem with me, how we can add two tables by using join and get the result as well as search.

**Comment by basicAngular** [comment]
@Manupreeta94  have you find any solution?

**Comment by basicAngular** [comment]
No, finally I have changed, doing it custom a lot of work, but at least it's working fine.
